docs/designs/credential.md

```diff
@@ -27,11 +27,12 @@ It can be configured in `docfx.yml`:
 ```yml
 referenceToAResource: "https://contoso.com/path/file"
 secrets:
-  "https://contoso.com/path/file": "?sig={token}"
+  http:
```

Comment:
👍

Reply:
do we auto-detect if it is auth header or query by checking if it starts with "?"

Reply:
These short form only take it as query string. Seems hard to keep key-value pair info in headers into a single string here.